kclvm/tools/src/langserver/tests.rs

```diff
@@ -0,0 +1,163 @@
+use crate::langserver;
+use crate::langserver::LineWord;
+use kclvm_error::Position;
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    fn check_line_to_words(code: &str, expect: Vec<LineWord>) {
+        assert_eq!(langserver::line_to_words(code.to_string()), expect);
+    }
+
+    #[test]
+    fn test_line_to_words() {
+        let datas = vec![
+            "alice_first_name = \"alice\"",
+            "中lice_first_name = \"alice\"",
```

Comment:
KCL does not support Unicode yet, so it can not exist in identifier names

Reply:
I've deleted this case.